<launch>

  <node name="controller"
        pkg ="joy"
        type="joy_node"
        respawn="false"
        output="screen">
    <remap from="/joy"
             to="/hyperion/joy"/>
    <param name="dev" value="/dev/input/js0"/>
  </node>

 <node name="hyperion"
       pkg ="hyperion_line_tracker"
       type="hyperion_line_tracker"
       respawn="false"
       output="screen">
    <param name="kp"            type="double"      value="99.0"/>
    <param name="ki"            type="double"      value="0.0"/>
    <param name="kd"            type="double"      value="1.5"/>
    <param name="pid_freq"      type="double"      value="200"/>
    <param name="max_speed"     type="int"         value="100"/>
    <param name="turn_speed_saturation" type="int" value="-100"/>
    <param name="stop_button"   type="int"         value="5"/>
    <param name="pid_tunning"   type="int"         value="0"/>
    <param name="tunning_inc"   type="double"      value="0.1"/>
    <param name="pid_mean_size" type="int"         value="0"/>
  </node>

  <node name="ir"
        pkg ="infrarrojos"
        type="infrarrojos_sensor.py"
        respawn="false"
        output="screen">
    <remap from="/infrarrojos_topic"
           to="/hyperion/ir_error"/>
  </node>-->

  <node name="motor_driver"
        pkg ="controlador_motores"
        type="motor_driver.py"
        respawn="false"
        output="screen">
    <remap from="/motor_speeds"
           to="/hyperion/speeds"/>
  </node>


</launch>
